DataStudies/limitPlotScripts/set_limit.py

```diff
@@ -359,22 +373,68 @@
         obsLineLabel.Draw()
 
 # Legend and draw
-gStyle.SetLegendFont(42)
-legend = TLegend(0.60, 0.50, 0.91, 0.87, '')
-legend.SetHeader("95% CL upper limits")
+gStyle.SetLegendFont(62)
+legend = TLegend(0.5, 0.6, 0.92, 0.9, '')
+legend.SetHeader("95% CL Upper Limits")
 if not options.blind:
-    legend.AddEntry(g_limit, "Observed", "l")
-legend.AddEntry(g_mclimit, "Median expected","l")
-legend.AddEntry(g_error, "68% expected", "f")
-legend.AddEntry(g_error95, "95% expected", "f")
-legend.AddEntry(graphWP, "Theory HSCP "+cstr+"", "l")   # NOT GENERIC
+    legend.AddEntry(g_limit, "Observed Limit", "l")
+legend.AddEntry(g_error95, "Expected Limit #pm1#sigma, #pm2#sigma","f")
+# legend.AddEntry(g_error95, "#pm1#sigma, 2#sigma", "f")
+# legend.AddEntry(g_error95, "#pm2#sigma", "f")
+legend.AddEntry(graphWP, "#sigma^{"+options.xsorder+"}_{th}("+options.process+")#pm1#sigma", "l")   # NOT GENERIC
 
 legend.SetBorderSize(0)
 legend.SetFillStyle(0)
 legend.SetLineColor(0)
 
 legend.Draw("same")
 
+# this is to fake the green+yellow band in the legend
+tmpcolor = g_error.GetFillColor()
+tmpline = ROOT.TLine()
+tmpline.SetLineColor(tmpcolor)
+tmpline.SetLineWidth(22)
+tmpyposition = 0.75
```

Comment:
these will be shifted when the unblind option is true, we'll have to figure out a way to be general in both cases